rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -1879,30 +1858,25 @@ rmw_take_response(
 
   // Fill in the request_header
 
-  request_header->request_id.sequence_number =
-    rmw_zenoh_cpp::get_int64_from_attachment(&sample->attachment, "sequence_number");
+  rmw_zenoh_cpp::attachement_data_t attachment(z_sample_attachment(sample));
```

Comment:
```suggestion
  rmw_zenoh_cpp::attachment_data_t attachment(z_sample_attachment(sample));
```

Reply:
Fixed in https://github.com/ros2/rmw_zenoh/pull/276/commits/9f78ca46344677a0a36d9c6fd651ee8e620f51b0